### ISSUE TITLE
Export data with custom labels on Firefox - missing CRLF

### ISSUE THREAD

**Issue by estellepicq** [LCluber/CSVx.js#5]
**Description**
Custom label row and first data row are not separated when exporting csv on Firefox.

- Here is the csv file content produced with Firefox:

```
﻿"First name";"Last name";"Born";"Died""Galileo";"Galiléi";"1564";"1642"
"Nikola";"Tesla";"1856";"1943"
"Albert";"Einstein";"1879";"1955"
```

- Here with Chrome:
```
﻿"First name";"Last name";"Born";"Died"
"Galileo";"Galiléi";"1564";"1642"
"Nikola";"Tesla";"1856";"1943"
"Albert";"Einstein";"1879";"1955"
```

**Code**
```
var customLabels = ['First name', 'Last name', 'Born', 'Died'];
exportButton.addEventListener('click', function () {
  CSVx.Export.data('scientists', array, {
    separator: ';',
    customLabels: customLabels
  });
});
```

**Comment by LCluber** [comment]
I will start testing the different types of data in createLabels(data: Object[]|string[]) method and see if it behaves like createCustomLabels() one way or another

**Comment by LCluber** [comment]
Fixed with release v0.1.11
